packages/@aws-cdk/aws-glue/README.md

```diff
@@ -225,7 +225,24 @@ new glue.Table(this, 'MyTable', {
 });
 ```
 
-By default, an S3 bucket will be created to store the table's data and stored in the bucket root. You can also manually pass the `bucket` and `s3Prefix`:
+Glue tables can be configured to contain user-defined properties, to describe the physical storage of table data, through the `storageParameters` property:
+
+```ts
+declare const myBucket: s3.Bucket;
+declare const myDatabase: glue.Database;
+new glue.Table(this, 'MyTable', {
+  database: myDatabase,
+  columns: [{
+    name: 'col1',
+    type: glue.Schema.STRING,
+  }],
+  dataFormat: glue.DataFormat.JSON,
+  storageParameters: {
+    'skip.header.line.count': 1,
+    'separatorChar': ',',
```

Comment:
is there a list of parameters that Glue recognizes? Can you input an invalid key? eg if I do:

```
storageParameters: {
  foo: 'bar'
}
```
is that legal? Also please don't use the quotes around the keys:

```suggestion
    'skip.header.line.count': 1,
    separatorChar: ',',
```

Reply:
The above is legal, but it will not have an effect on the glue table. The description of the parameter denotes this.